src/entity/user/Customer.java

```diff
@@ -0,0 +1,322 @@
+package entity.user;
+
+import entity.booking.Booking;
+import entity.hotel.Hotel;
+import entity.payment.Payment;
+import entity.payment.PaymentMethod;
+import entity.payment.PaymentStatus;
+import entity.review.Review;
+import entity.room.Room;
+import entity.room.RoomStatus;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Customer extends User  {
+    private static Integer count = 2;
+    private UserDocument userDocument;
+    private String address;
+    private String telephone;
+    private Set<Booking> bookingSet;
+    private Set<Payment> paymentSet;
+
+    public Customer(String firstName, String lastName, UserDocument userDocument, String address, String telephone, String username, String password, String email) {
+        super(count, firstName, lastName, username, password, email);
+        this.userDocument = userDocument;
+        this.address = address;
+        this.telephone = telephone;
+        bookingSet = new HashSet<>();
+        paymentSet = new HashSet<>();
+        count++;
+    }
+
+    public UserDocument getUserDocument() {
+        return userDocument;
+    }
+
+    public void setUserDocument(UserDocument userDocument) {
+        this.userDocument = userDocument;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public String getTelephone() {
+        return telephone;
+    }
+
+    public void setTelephone(String telephone) {
+        this.telephone = telephone;
+    }
+
+    public Set<Booking> getBookingSet() {
+        return bookingSet;
+    }
+
+    public void setBookingSet(Set<Booking> bookingSet) {
+        this.bookingSet = bookingSet;
+    }
+
+    public Set<Payment> getPaymentSet() {
+        return paymentSet;
+    }
+
+    public void setPaymentSet(Set<Payment> paymentSet) {
+        this.paymentSet = paymentSet;
+    }
+
+
+    @Override
+    public String getUserType() {
+        return "Customer";
+    }
+
+    public void viewProfile(){
+        System.out.println(this);
+    }
+
+    public void viewHotelServices(){
+        Hotel hotel = Hotel.getHotelInstance();
+        System.out.println(hotel.getHotelServices());
+    }
+
+    public void viewAvailableRooms(){
+        Hotel hotel = Hotel.getHotelInstance();
+        System.out.println(hotel.getRoomList().stream().filter(x -> x.getRoomStatus() == RoomStatus.AVAILABLE).toList());
+    }
```

Comment:
line 95 does not compile on my laptop - does it work on yours? 
fix: System.out.println(hotel.getRoomList().stream().filter(x -> x.getRoomStatus() == RoomStatus.AVAILABLE).collect(Collectors.toList())); - you need to collect the elements of the stream generated by the filter method in a List using Collectors

Reply:
La mine compileaza tot, dar voi rezolva. Multumesc!